wp-content/themes/eighshift-task/src/blocks/custom/posts/assets/load-more.js

```diff
@@ -0,0 +1,61 @@
+import '@babel/polyfill';
```

Comment:
you put this globaly in application.js or some thing like that because it apples to all the code and not to only this component